package.json

```diff
@@ -15,23 +18,25 @@
     "@types/color": "^3.0.6",
     "@types/jest": "^29.5.12",
     "@types/react": "^18.3.3",
-    "eslint": "^8.57.0",
+    "eslint": "^9.8.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-import": "^2.29.1",
     "eslint-plugin-jsx-a11y": "^6.9.0",
     "eslint-plugin-observation": "https://github.com/observation/eslint-rules.git",
-    "eslint-plugin-prettier": "^5.1.3",
-    "eslint-plugin-react": "^7.34.3",
+    "eslint-plugin-prettier": "^5.2.1",
+    "eslint-plugin-react": "^7.35.0",
     "eslint-plugin-react-hooks": "^4.6.2",
     "eslint-plugin-react-native": "^4.1.0",
+    "globals": "^15.8.0",
     "jest": "^29.7.0",
-    "prettier": "^3.3.2",
+    "prettier": "^3.3.3",
     "react": "18.2.0",
     "react-native": "0.74.3",
     "react-native-render-html": "^6.3.4",
     "react-test-renderer": "18.2.0",
     "ts-jest": "^29.2.2",
-    "typescript": "5.0.4"
+    "typescript": "5.0.4",
+    "typescript-eslint": "^7.18.0"
```

Comment:
Deze heeft inmiddels versie 8, maar ik weet nou niet of dit makkelijk is om te gebruiken